src/socketio.quiver.js

```diff
@@ -208,7 +234,7 @@ QuiverSocialProvider.count_ = function(obj) {
 };
 
 QuiverSocialProvider.prototype.getClientId_ = function() {
-  return this.configuration_.self.id + ':' + this.clientSuffix_;
+  return this.configuration_.self.id + '#' + this.clientSuffix_;
```

Comment:
':' can appear inside the id (and suffix), so it's not a great delimiter.  '#' is safe (I think...).